test/e2e/config/docker.yaml

```diff
@@ -29,8 +29,13 @@ providers:
 - name: cluster-api
   type: CoreProvider
   versions:
-  - name: v0.4.0
-  # Use manifest from source files
+  - name: v0.3.16 # latest published release
+    value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.16/core-components.yaml"
```

Comment:
Will adding this versions here have side effects for other tests?

Reply:
No, because other test does not specify a version, so the latest available will be used (which is v1alpha4 as expected)